docs/DafnyRef/UserGuide.md

```diff
@@ -31,21 +31,22 @@ are described on the Dafny wiki:
 They are not repeated here to avoid replicating information that
 easily becomes inconsistent and out of date.
 The dafny tool can also be installed using `dotnet tool install --global dafny`
-(given that `dotnet` is already installed on your system).
+(presuming that `dotnet` is already installed on your system).
 
-Most users will find it most convenient to install the pre-built Dafny binaries available on the project release site.
+Most users will find it most convenient to install the pre-built Dafny binaries available on the project release site or in dotnet.
 As is typical for Open Source projects, dafny can also be built directly from the source files maintained in the github project.
 
 Current and past Dafny binary releases can be found at
 [https://github.com/dafny-lang/dafny/releases](https://github.com/dafny-lang/dafny/releases) for each supported platform.
 Each release is a .zip file with a name combining the release name and the
-platform. Current platforms are Windows 10, Ubuntu 16ff, and MacOS 10.14ff.
+platform. Current platforms are Windows 11, Ubuntu 20ff, and MacOS 10.14ff.
```

Comment:
What do the "ff" suffixes mean?

Reply:
And following. But I revised it to be clearer. Besides ff has lots more meanings today. (friends forever?)